src/Sylius/Behat/Context/Ui/EmailContext.php

```diff
@@ -166,11 +188,24 @@ public function recipientShouldReceiveNoEmails(string $recipient): void
         Assert::false($this->emailChecker->hasRecipient($recipient));
     }
 
+    /**
+     * @Then /^only one email should have been sent to "([^"]*)"$/
+     */
+    public function onlyOneEmailShouldHaveBeenSentTo(string $recipient): void
+    {
+        Assert::eq($this->emailChecker->countMessagesTo($recipient), 1);
+    }
+
     private function assertEmailContainsMessageTo(string $message, string $recipient): void
     {
         Assert::true($this->emailChecker->hasMessageTo($message, $recipient));
     }
 
+    private function assertEmailDoesNotContainMessageTo(string $message, string $recipient): void
```

Comment:
```suggestion
    private function assertEmailDoesNotContainsMessageTo(string $message, string $recipient): void
```

Reply:
✅ It doesn't contain
❌ It doesn't contain**s**

The function's name is correct :D.